python/mxnet/initializer.py

```diff
@@ -81,6 +83,13 @@ def _init_bias(self, _, arr):
     def _init_gamma(self, _, arr):
         arr[:] = 1.0
 
+    def _init_scale(self, name, arr):
+        try:
+            s = float(name[:-6].split('_')[-1])
```

Comment:
this seems like a very specific thing.
Use mx.init.Mixed when initializing weight instead of hacking the default initializer